adoc/Release-Notes.adoc

```diff
@@ -2,7 +2,7 @@
 ifdef::env-github[]
 :suse: SUSE
 :product: {suse} Cloud Application Platform
-:version: 1.3.1
+:version: 1.4
```

Comment:
This value will be ignored in the published document -- please fix the value in `adoc/entities.adoc` and then run `make update-entries`.

Reply:
Done via https://github.com/SUSE/cap-release-notes/pull/64